src/service.py

```diff
@@ -28,12 +28,23 @@
 
 from bayes_optuna import optuna_hpo
 
+# Importing socket and os library
+import socket
+import os
+
 autotune_object_ids = {}
+search_space_json = []
 
 api_endpoint = "/experiment_trials"
-host_name = "localhost"
 server_port = 8085
 
+# Default values
+if os.getenv("N_TRIALS") == None :
+    os.environ['N_TRIALS'] = '10'
+
+if os.getenv("N_JOBS") == None :
+    os.environ['N_JOBS'] = '1'
+
```

Comment:
Can we log these values so that we are aware what are the values for N_JOBS and N_TRIALS

Reply:
Done